docs/source/development/browser.rst

```diff
@@ -34,6 +34,8 @@ AngularJS 1.3 has dropped support for IE8. Read more about it on this `blog <htt
 
 The AngularJS project currently supports and will attempt to fix bugs for IE9 and above. Their continuous integration server runs all the tests against IE9, IE10, and IE11. See `Travis CI <https://travis-ci.org/angular/angular.js>`_ and `ci.angularjs.org <https://ci.angularjs.org>`_.
 
+However, MOTECH supports IE11 and above.
```

Comment:
@martokarski @afijal Let's add an explanation based on **Supported browsers** paragraph from this already merged [PR on the Motech UI](https://github.com/motech/motech-ui/pull/32/files) to keep both documentations as similar as possible